sanitycheck fails on tests/benchmarks/latency_measure/benchmark.kernel.latency

`./scripts/sanitycheck -s tests/benchmarks/latency_measure/benchmark.kernel.latency -p mec15xxevb_assy6853`

Gets:

```
Memory region         Used Size  Region Size  %age Used
           FLASH:       30904 B       224 KB     13.47%
            SRAM:       12960 B        32 KB     39.55%
        IDT_LIST:         216 B         2 KB     10.55/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(coop_ctx_switch.c.obj): in function `timing_counter_get':
/home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: /home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(coop_ctx_switch.c.obj): in function `timing_cycles_get':
/home/galak/git/zephyr/include/timing/timing.h:98: undefined reference to `soc_timing_cycles_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(coop_ctx_switch.c.obj): in function `timing_cycles_to_ns_avg':
/home/galak/git/zephyr/include/timing/timing.h:149: undefined reference to `soc_timing_cycles_to_ns_avg'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(int_to_thread.c.obj): in function `timing_counter_get':
/home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(int_to_thread.c.obj): in function `timing_cycles_get':
/home/galak/git/zephyr/include/timing/timing.h:98: undefined reference to `soc_timing_cycles_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(int_to_thread.c.obj): in function `timing_cycles_to_ns':
/home/galak/git/zephyr/include/timing/timing.h:131: undefined reference to `soc_timing_cycles_to_ns'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(int_to_thread.c.obj): in function `timing_counter_get':
/home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(int_to_thread_evt.c.obj): in function `timing_counter_get':
/home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: /home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(int_to_thread_evt.c.obj): in function `timing_cycles_get':
/home/galak/git/zephyr/include/timing/timing.h:98: undefined reference to `soc_timing_cycles_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(int_to_thread_evt.c.obj): in function `timing_cycles_to_ns':
/home/galak/git/zephyr/include/timing/timing.h:131: undefined reference to `soc_timing_cycles_to_ns'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(main.c.obj): in function `timing_freq_get_mhz':
/home/galak/git/zephyr/include/timing/timing.h:165: undefined reference to `soc_timing_freq_get_mhz'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(mutex_lock_unlock.c.obj): in function `timing_counter_get':
/home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: /home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(mutex_lock_unlock.c.obj): in function `timing_cycles_get':
/home/galak/git/zephyr/include/timing/timing.h:98: undefined reference to `soc_timing_cycles_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(mutex_lock_unlock.c.obj): in function `timing_cycles_to_ns_avg':
/home/galak/git/zephyr/include/timing/timing.h:149: undefined reference to `soc_timing_cycles_to_ns_avg'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(mutex_lock_unlock.c.obj): in function `timing_counter_get':
/home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: /home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(mutex_lock_unlock.c.obj): in function `timing_cycles_get':
/home/galak/git/zephyr/include/timing/timing.h:98: undefined reference to `soc_timing_cycles_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(mutex_lock_unlock.c.obj): in function `timing_cycles_to_ns_avg':
/home/galak/git/zephyr/include/timing/timing.h:149: undefined reference to `soc_timing_cycles_to_ns_avg'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(sema_test_signal_release.c.obj): in function `timing_counter_get':
/home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: /home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: /home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(sema_test_signal_release.c.obj): in function `timing_cycles_get':
/home/galak/git/zephyr/include/timing/timing.h:98: undefined reference to `soc_timing_cycles_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(sema_test_signal_release.c.obj): in function `timing_cycles_to_ns':
/home/galak/git/zephyr/include/timing/timing.h:131: undefined reference to `soc_timing_cycles_to_ns'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(sema_test_signal_release.c.obj): in function `timing_counter_get':
/home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(sema_test_signal_release.c.obj): in function `timing_cycles_get':
/home/galak/git/zephyr/include/timing/timing.h:98: undefined reference to `soc_timing_cycles_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(sema_test_signal_release.c.obj): in function `timing_cycles_to_ns':
/home/galak/git/zephyr/include/timing/timing.h:131: undefined reference to `soc_timing_cycles_to_ns'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(sema_test_signal_release.c.obj): in function `timing_counter_get':
/home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: /home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(sema_test_signal_release.c.obj): in function `timing_cycles_get':
/home/galak/git/zephyr/include/timing/timing.h:98: undefined reference to `soc_timing_cycles_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(sema_test_signal_release.c.obj): in function `timing_cycles_to_ns_avg':
/home/galak/git/zephyr/include/timing/timing.h:149: undefined reference to `soc_timing_cycles_to_ns_avg'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(sema_test_signal_release.c.obj): in function `timing_counter_get':
/home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: /home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(sema_test_signal_release.c.obj): in function `timing_cycles_get':
/home/galak/git/zephyr/include/timing/timing.h:98: undefined reference to `soc_timing_cycles_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(sema_test_signal_release.c.obj): in function `timing_cycles_to_ns_avg':
/home/galak/git/zephyr/include/timing/timing.h:149: undefined reference to `soc_timing_cycles_to_ns_avg'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(thread.c.obj): in function `timing_counter_get':
/home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: /home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: /home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: /home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: /home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(thread.c.obj):/home/galak/git/zephyr/include/timing/timing.h:76: more undefined references to `soc_timing_counter_get' follow
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(thread.c.obj): in function `timing_cycles_get':
/home/galak/git/zephyr/include/timing/timing.h:98: undefined reference to `soc_timing_cycles_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(thread.c.obj): in function `timing_cycles_to_ns':
/home/galak/git/zephyr/include/timing/timing.h:131: undefined reference to `soc_timing_cycles_to_ns'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(thread.c.obj): in function `timing_cycles_get':
/home/galak/git/zephyr/include/timing/timing.h:98: undefined reference to `soc_timing_cycles_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(thread.c.obj): in function `timing_cycles_to_ns':
/home/galak/git/zephyr/include/timing/timing.h:131: undefined reference to `soc_timing_cycles_to_ns'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(thread.c.obj): in function `timing_cycles_get':
/home/galak/git/zephyr/include/timing/timing.h:98: undefined reference to `soc_timing_cycles_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(thread.c.obj): in function `timing_cycles_to_ns':
/home/galak/git/zephyr/include/timing/timing.h:131: undefined reference to `soc_timing_cycles_to_ns'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(thread.c.obj): in function `timing_cycles_get':
/home/galak/git/zephyr/include/timing/timing.h:98: undefined reference to `soc_timing_cycles_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(thread.c.obj): in function `timing_cycles_to_ns':
/home/galak/git/zephyr/include/timing/timing.h:131: undefined reference to `soc_timing_cycles_to_ns'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(thread.c.obj): in function `timing_counter_get':
/home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: /home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(thread.c.obj): in function `timing_cycles_get':
/home/galak/git/zephyr/include/timing/timing.h:98: undefined reference to `soc_timing_cycles_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(thread.c.obj): in function `timing_cycles_to_ns':
/home/galak/git/zephyr/include/timing/timing.h:131: undefined reference to `soc_timing_cycles_to_ns'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(thread_switch_yield.c.obj): in function `timing_counter_get':
/home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: /home/galak/git/zephyr/include/timing/timing.h:76: undefined reference to `soc_timing_counter_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(thread_switch_yield.c.obj): in function `timing_cycles_get':
/home/galak/git/zephyr/include/timing/timing.h:98: undefined reference to `soc_timing_cycles_get'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: ../app/libapp.a(thread_switch_yield.c.obj): in function `timing_cycles_to_ns_avg':
/home/galak/git/zephyr/include/timing/timing.h:149: undefined reference to `soc_timing_cycles_to_ns_avg'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: subsys/timing/libsubsys__timing.a(timing.c.obj): in function `timing_init':
/home/galak/git/zephyr/subsys/timing/timing.c:24: undefined reference to `soc_timing_init'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: subsys/timing/libsubsys__timing.a(timing.c.obj): in function `timing_start':
/home/galak/git/zephyr/subsys/timing/timing.c:41: undefined reference to `soc_timing_start'
/opt/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: subsys/timing/libsubsys__timing.a(timing.c.obj): in function `timing_stop':
/home/galak/git/zephyr/subsys/timing/timing.c:56: undefined reference to `soc_timing_stop'
collect2: error: ld returned 1 exit status
%
```

This is due to commit:
commit 9be37553eee22bc16f8740fa2e2ad17b7bccd509
Author: Daniel Leung <daniel.leung@intel.com>
Date:   Wed Oct 14 11:15:40 2020 -0700

    timing: do not repeatedly do init()/start()/stop()

--- a/soc/arm/microchip_mec/mec1501/Kconfig.defconfig.series
+++ b/soc/arm/microchip_mec/mec1501/Kconfig.defconfig.series
@@ -22,7 +22,7 @@ config MCHP_XEC_RTOS_TIMER
 	default y
 
 config SOC_HAS_TIMING_FUNCTIONS
-	default y
+	default y if !CORTEX_M_DWT
 
 config ARCH_HAS_CUSTOM_BUSY_WAIT
 	default y

